pyproject.toml

```diff
@@ -54,6 +57,9 @@ documentation = "https://mnelab.readthedocs.io/en/latest/"
 repository = "https://github.com/cbrnr/mnelab"
 changelog = "https://github.com/cbrnr/mnelab/blob/main/CHANGELOG.md"
 
+[tool.hatch.version]
+path = "src/mnelab/__init__.py"
```

Comment:
why is this needed?

Reply:
Because that's where the version is defined. This will become obsolete when I switch to `hatch-vcs` though.